src/event-handler.js

```diff
@@ -6,7 +6,9 @@ function EventHandler(common) {
 }
 
 EventHandler.prototype.logEvent = function(event) {
-    if (optimizelyWebXEvents.events[event.EventName]) {
+    var self = this;
```

Comment:
since the common.useFullStack is not in a new context, you can remove this self, and just use this

Reply:
A little confused on this one. If I remove `self`, and use `this` in event-handler, instead, things break.

Reply:
not sure why it would.  i just made the change locally and everything still passed.

this screenshot shows a debugger in the event handler, it shows that `self.common.useFullStack` and  `this.common.useFullStack` are the same here:

![image](https://user-images.githubusercontent.com/5377436/90786492-9e4b5200-e2d1-11ea-998d-b7d117f01f4a.png)